cosmwasm/contracts/v1/compute-tests/bench-contract/src/contract.rs

```diff
@@ -44,17 +57,87 @@ pub fn execute(
             bench_read_large_key_from_storage(deps, 2)
         }
         ExecuteMsg::BenchWriteLargeItemToStorage { .. } => bench_write_large_storage_key(deps, 2),
+        ExecuteMsg::CreateViewingKey {} => try_create_key(deps, env, info),
+        ExecuteMsg::SetViewingKey { key, .. } => try_set_key(deps, info, key),
     };
 
     Ok(Response::default())
 }
 
 #[entry_point]
-pub fn query(_deps: Deps, _env: Env, _msg: QueryMsg) -> StdResult<Binary> {
-    Ok(Binary::default())
+pub fn query(deps: Deps, _env: Env, msg: QueryMsg) -> StdResult<Binary> {
+    match msg {
+        QueryMsg::WithPermit { permit, query } => permit_queries(deps, permit, query),
+        _ => viewing_keys_queries(deps, msg),
+    }
 }
 
 #[entry_point]
 pub fn reply(_deps: DepsMut, _env: Env, _reply: Reply) -> StdResult<Response> {
     Ok(Response::default())
 }
+
+pub fn try_create_key(deps: DepsMut, env: Env, info: MessageInfo) -> StdResult<()> {
```

Comment:
This function shouldn't return anything and should be named "create_key" as the result is always Ok(()) in this case